src/BenchmarksApps/Antiforgery/Program.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Antiforgery;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Logging.ClearProviders();
+builder.Services.AddAntiforgery(options => options.HeaderName = "XSRF-TOKEN");
+
+var app = builder.Build();
+app.UseAntiforgery();
+
+app.MapGet("/", () => Results.Ok("hello world!"));
+
+// GET https://localhost:55471/auth
+app.MapGet("/auth", (HttpContext ctx, IAntiforgery antiforgery) =>
+{
+    var token = antiforgery.GetAndStoreTokens(ctx);
+    ctx.Response.Headers.Append("XSRF-TOKEN", token.RequestToken!);
+    return Results.Ok();
+});
+
+// POST https://localhost:55471/validateToken
+app.MapPost("/validateToken", async (HttpContext ctx, IAntiforgery antiforgery) =>
```

Comment:
Would be great to expose another endpoint that does the same work (return 200?) but not doing the validation, so you can have a baseline to compare the cost of validation

Reply:
added the `/noOp` endpoint with such definition:
```
app.MapPost("/noOp", (HttpContext ctx, IAntiforgery antiforgery) => Results.Ok());
```

and added the benchmark scenario to compare that against only token-generation or token-validation:
```
  - displayName: Antiforgery No Operation
    arguments: --scenario antiforgery-noop $(antiforgeryJobs) --property scenario=AntiforgeryNoOp
  - displayName: Antiforgery Token Validation
    arguments: --scenario antiforgery-validation $(antiforgeryJobs) --property scenario=AntiforgeryTokenValidation
  - displayName: Antiforgery Token Generation
    arguments: --scenario antiforgery-generation $(antiforgeryJobs) --property scenario=AntiforgeryTokenGeneration

```